mappings/net/minecraft/data/DataGenerator.mapping

```diff
@@ -1,3 +1,28 @@
 CLASS net/minecraft/unmapped/C_nditonqu net/minecraft/data/DataGenerator
+	FIELD f_hmnncbno providerIds Ljava/util/Set;
+	FIELD f_icwqrlpq ignoreCache Z
+	FIELD f_jnyygmxo providersToRun Ljava/util/Map;
+	FIELD f_luxffyyp output Lnet/minecraft/unmapped/C_ugkmwocs;
+	FIELD f_nbnmbvtz rootOutputPath Ljava/nio/file/Path;
+	FIELD f_qgqimoxw version Lnet/minecraft/unmapped/C_xyxorzsk;
 	FIELD f_zacxiejs LOGGER Lorg/slf4j/Logger;
+	METHOD <init> (Ljava/nio/file/Path;Lnet/minecraft/unmapped/C_xyxorzsk;Z)V
+		ARG 1 rootOutputPath
+		ARG 2 version
+		ARG 3 ignoreCache
 	METHOD m_bejppmfd run ()V
+	METHOD m_hvhbaasw createVanillaPack (Z)Lnet/minecraft/unmapped/C_nditonqu$C_joaphpus;
+		ARG 1 shouldRun
+	METHOD m_upsqvizx createDatapack (ZLjava/lang/String;)Lnet/minecraft/unmapped/C_nditonqu$C_joaphpus;
```

Comment:
Hm, shouldn't this be `createDataPack`?
```suggestion
	METHOD m_upsqvizx createDataPack (ZLjava/lang/String;)Lnet/minecraft/unmapped/C_nditonqu$C_joaphpus;
```

Reply:
yes, the reason it was one word was because I searched `Datapack` to check convention and the first thing I saw was `DatapackCommand`, which is a special case

Reply:
ah, i see